eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -15,16 +16,19 @@ jobs:
         VCPKG_DEFAULT_TRIPLET: 'x64-linux'
         CC: '/usr/bin/gcc-8'
         CXX: '/usr/bin/g++-8'
+        BuildArgs: '-j 10'
```

Comment:
Can you add a comment on what -j 10 does.  Its not easy to search for that build arg and what it does.

Reply:
this is just to speed up Linux builds
It add parallel compilation (10 threads to build).
It reduces building time when building samples and tests, from 4 min to 2 min.

It's not really related to code cov. I can remove it for another PR maybe.